.github/CODEOWNERS

```diff
@@ -1,4 +1 @@
-*         @idreyn @mathcolo @devinmatte
-/.github/ @idreyn @mathcolo @devinmatte @nathan-weinberg
-deploy.sh @idreyn @mathcolo @devinmatte @nathan-weinberg
-.flake8   @idreyn @mathcolo @devinmatte @nathan-weinberg
+* @idreyn @mathcolo @devinmatte @skaplan-dev @nathan-weinberg
```

Comment:
Can we add @transitmatters/labs-leads so the group always pings the current leads even as they change